lib/tree.rb

```diff
@@ -1,38 +1,60 @@
 class NoApplesError < StandardError; end
 
-class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+RANDOM = Random.new;
+
+class Tree
+  attr_reader :height, :age, :apples, :alive
 
   def initialize
+    @age = 0
+    @height = 1
+    @apples = []
+    @alive = true
   end
 
   def age!
+    @age += 1
+    if @alive
```

Comment:
I was wondering about that. It seemed like a lot of the examples I was seeing used the variable directly. I will change to the accessors, which I prefer.